Documentation/DSCResources/VMHostPowerPolicy/VMHostPowerPolicy.md

```diff
@@ -0,0 +1,51 @@
+# VMHostPowerPolicy
+
+## Parameters
+
+| Parameter | Attribute | DataType | Description | Allowed Values |
+| --- | --- | --- | --- | --- |
+| **Server** | Key | string | Name of the Server we are trying to connect to. The Server can be a vCenter or ESXi. ||
+| **Name** | Key | string | Name of the VMHost to configure. ||
+| **Credential** | Mandatory | PSCredential | Credentials needed for connection to the specified Server. ||
+| **PowerPolicy** | Mandatory | PowerPolicy | The key for the VMHost Power Management Policy. | HighPerformance, Balanced, LowPower, Custom |
```

Comment:
Update description not to contain key word

Reply:
Done.